graylog2-web-interface/src/components/sidecars/sidecars/SidecarStatus.jsx

```diff
@@ -70,6 +76,12 @@ const SidecarStatus = createReactClass({
           statusMessage = status.message;
           statusClass = 'text-danger';
           statusBadge = <i className="fa fa-warning fa-fw" />;
+
+          if (status.verbose_message) {
+            verboseButton = <Button bsStyle="link" bsSize="sm" onClick={() => {
```

Comment:
There is a couple of small issues here:

1. We wrap multi-line components in parenthesis (https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-wrap-multilines.md)
1. When props are too long, we only put one per line (https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-max-props-per-line.md)
1. The size of the button is a bit too big, using `xs` makes it look a bit better imho
1. In this case you don't need to wrap the `onClick` function handle with braces (it's a single line), and that improves readability
1. I think the text in the button reads better without parenthesis and without ellipsis at the end

Before:
<img width="594" alt="before" src="https://user-images.githubusercontent.com/716185/47731191-e6fccb80-dc63-11e8-96bc-46d336a04406.png">

After:
<img width="592" alt="after" src="https://user-images.githubusercontent.com/716185/47731190-e6fccb80-dc63-11e8-99ea-4649055f044e.png">


I couldn't do a multi-line suggestion, so here's the code I used for the after screenshot:

```jsx
            verboseButton = (
              <Button bsStyle="link"
                      bsSize="xs"
                      onClick={() => this._onShowVerbose(collector.name, status.verbose_message)}>
                Show Details
              </Button>
            );
```